Doc/library/curses.rst

```diff
@@ -219,8 +227,8 @@ The module :mod:`curses` defines the following functions:
 
 .. function:: getsyx()
 
-   Return the current coordinates of the virtual screen cursor in y and x.  If
-   leaveok is currently true, then -1,-1 is returned.
+   Return the current coordinates of the virtual screen cursor as a tuple
+   ``(y, x)``.  If **leaveok** is currently true, then ``(-1, -1)`` is returned.
```

Comment:
**leaveok** is a name of the window option. It is changed by the method `window.leaveok()`. How better to reword this?